.github/workflows/gradle.yml

```diff
@@ -0,0 +1,40 @@
+# This workflow will build a Java project with Gradle
+# For more information see: https://help.github.com/actions/language-and-framework-guides/building-and-testing-java-with-gradle
+
+name: Java CI with Gradle
+
+on: [push,pull_request]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up JDK 1.11
+        uses: actions/setup-java@v1
+        with:
+          java-version: 1.11
+
+      - name: Grant execute permission for gradlew
+        run: chmod +x gradlew
+
+      - name: install jq
```

Comment:
I dont think this is necessary anymore as it was used for last version of codacy action

Reply:
Ok, I will remove it in next PR.